Source/PINDiskCache.m

```diff
@@ -858,7 +862,7 @@ - (id)objectForKeyedSubscript:(NSString *)key
                   [self lock];
                       [[NSFileManager defaultManager] removeItemAtPath:[fileURL path] error:&error];
                   [self unlock];
-                  PINDiskCacheError(error);
+                  PINDiskCacheException(exception);
```

Comment:
In this case should we not also print the error besides the exception?

```
PINDiskCacheError(error);
PINDiskCacheException(exception);
```

Reply:
Probably could, error will most likely be nil most of the time. Removing the file should succeed in most cases.

Reply:
I agree with @maicki we should leave the PINDiskCacheError. We should probably move the PINDiskCacheException to the top of this scope as well?

Reply:
If we make the PINDiskCacheException do an assert instead, won't it crash before its able to remove the file and print the error? Or perhaps the file shouldn't be removed if it crashes in debug? Feel like we want to remove it in every case though.

Reply:
Yeah that sounds good to me. I've actually already made the changes for #3. See below.